content/millennium/r4/individuals/related-person.md

```diff
@@ -0,0 +1,105 @@
+---
+title: RelatedPerson | R4 API
+---
+
+# RelatedPerson
+
+* TOC
+{:toc}
+
+## Overview
+
+The RelatedPerson resource provides information about a person who is involved in the care of a patient but has no formal responsibility. RelatedPersons typically have a personal relationship with the patient such as a spouse, relative, friend, guardian, or attorney. RelatedPersons are commonly a source of information about a patient.
+
+The distinction between Practitioner and RelatedPerson is a practitioner cares for multiple patients on behalf of a healthcare facility where a RelatedPerson performs care tasks for a specific patient and is not associated with any healthcare facility.
+
+* [RelatedPerson id](http://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
```

Comment:
The following fields are returned if valued:

Reply:
https://github.com/cerner/fhir.cerner.com/pull/254/commits/a6cc265db68cabf4cb1d9abc1dec64f71d1c0b86#diff-b47a4cc256449f5aa96d433a01e1ae01R16

Reply:
identifier=urn:oid:2.16.840.1.113883.3.13.6|RANDES011E8442-2E86-4A00-8394-EFC6A3434A8F